src/utils/functions/formatNumber.ts

```diff
@@ -1,8 +1,9 @@
 export const formatNumber = (num: number) => {
-  if (num > 999 && num < 1000000) {
-    return (num / 1000).toFixed(0) + 'k'
+var str = 'k'
```

Comment:
we can use const variable

Reply:
> we can use the const variable and a meaningful name 🙇🏻

Hi @sasicodes, updated the variable name to `attribute` , pls let me know if we can go ahead with this?